components/bl-carousel-component/src/helpers.js

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export const useClassNamesItem = (
+  index,
+  currentImg,
+  nextCurrentImage,
+  isNextAnimation,
+  isPrevAnimation,
+  typeAnimation
```

Comment:
Can this be fit in one line?

Reply:
then the string will exceed 120 characters